docs/proposals/kubelet-qos.md

```diff
@@ -0,0 +1,142 @@
+# Quality of Service in Kubernetes

```

Comment:
Please rename the doc, also, after comments are addressed (so we don't lose them).